src/qt/transactiontablemodel.cpp

```diff
@@ -276,8 +276,7 @@ void TransactionTableModel::updateAmountColumnTitle()
 
 void TransactionTableModel::updateTransaction(const QString &hash, int status, bool showTransaction)
 {
-    uint256 updated;
-    updated.SetHexDeprecated(hash.toStdString());
+    Txid updated = Txid::FromHex(hash.toStdString()).value();
```

Comment:
This seems reasonable: `updateTransaction` is called from `TransactionNotification::invoke`, which creates the string using `uint256::GetHex`.

Reply:
This use of `SetHexDeprecated` could be eliminated by changing the type of the first parameter from `const QString&` to `const uint256&`. See [this commit](https://github.com/hebasto/bitcoin/commit/01b84ba9a78597d9f7e1ab0c76fda0c56b32980f) in [this branch](https://github.com/hebasto/bitcoin/commits/250409-pr32237-AMENDED/).